apps/sms/js/thread_ui.js

```diff
@@ -734,10 +736,10 @@ var ThreadUI = {
       // Recipients will exist for draft messages in threads
       // Otherwise find them from draft recipient numbers
       draft.recipients.forEach(function(number) {
-        Contacts.findByAddress(number, function(records) {
-          if (records.length) {
+        Contacts.findByAddress(number).then(function(contacts) {
+          if (contacts.length) {
```

Comment:
Can we potentially get ```null reference``` exception here with the current patch(as before patch as well)?